### ISSUE TITLE
Plans to Godot 4.3?

### ISSUE THREAD

**Issue by diogovilarbr** [kyboon/ModularSpriteAnimationFactory#1]
Any chance to update that wonderfull project to Godot 4.3?

**Comment by kyboon** [comment]
Yes I do plan to do that. Currently I'm on vacation so it won't be done until September.

**Comment by kyboon** [comment]
Hi I just tested this project on Godot 4.3, and it seems to be working without any issue. Is there any reason you're suggesting me to "update it to Godot 4.3"? Did you encountered any problem?

If not, I'll close this issue.

**Comment by ceseuskadaver** [comment]
Hi! I noticed that the addon doesn't seem visible in the built-in Asset Library in Godot, while it is visible in the web version of the Asset Library. I'm not sure if this what Diago meant, but it is strange since other addons for Godot 4.0 are visible.

**Comment by kyboon** [comment]
Well that's weird. I can see it in the built-in Asset Library in Godot 4.3.
![image](https://github.com/user-attachments/assets/70e17850-f66b-4caa-a910-768e87cac048)
